.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: ci
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+env:
+  cache-version: v1
+
+jobs:
+  cache_bundle_install:
+    # see: https://github.com/actions/virtual-environments
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        ruby-version: ["3.1.1"]
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-ruby@v1
```

Comment:
NOTE: This action is deprecated and is no longer maintained.
Please, migrate to https://github.com/ruby/setup-ruby, which is being actively maintained.